t/01-pdk/08-response/11-exit.t

```diff
@@ -605,3 +605,59 @@ Content-Length: 19
 {"message":"hello"}
 --- no_error_log
 [error]
+
+
+
+=== TEST 23: response.exit() does not send body with gRPC
```

Comment:
I suggest at least three more test cases for the scenarios discussed above:
* what happens when you set the content-type explicitly but send a table
  * it will currently send `body.message` as payload, but I think it should send nothing
* what happens when you set the content-type explicitly but send a string
  * I think it should send the string as payload
* what happens when you don't set the content-type but send a string
  * I think it should send the string as payload